web/controllers/notes_controller.ex

```diff
@@ -21,12 +21,26 @@ defmodule Notox.NoteController do
   def create(conn, %{"note" => note_params}) do
     changeset = Note.changeset(%Note{}, note_params)
     case Repo.insert(changeset) do
-      {:ok, note} ->
+      {:ok, _note} ->
```

Comment:
I was faced with the question of using `_note` or just `_`. Thoughts on this? I was thinking since I know what is, acknowledging it might make it more understandable. 

I think I would use `_` if it could be two distinct things and I still didn't care about it. Like for example how `Repo.update/1` returns either a `Note` or a `changeset`.